src/components/LoanCards/FavoriteStar.vue

```diff
@@ -1,18 +1,52 @@
 <template>
-	<button
-		v-kv-track-event="['Lending', 'click-Favorite star', 'Favorite', loanId, loanId]"
-		@click.prevent.stop="toggleFavorite"
-	>
-		<kv-icon name="star" :from-sprite="true" class="icon-star" :class="{ 'is-favorite': isFavorite }" />
-	</button>
+	<!--
+	*
+	*
+	*
+	*
+	*
+		This is a old file, we don't want to use moving forward
+		if you're looking to implement loan bookmarking behavior
+		please use:
+			src/components/BorrowerProfile/LoanBookmark.vue
+	*
+	*
+	*
+	*
+	*-->
+	<span>
+		<button
+			v-if="!isFavorite"
+			class="tw-inline-flex tw-p-1 tw-cursor-pointer tw-bg-primary tw-rounded tw-m-0.5"
+			@click.prevent.stop="toggleFavorite()"
+			v-kv-track-event="['Lending', 'click-Favorite star', 'Favorite', loanId, loanId]"
+		>
+			<kv-material-icon
+				class="tw-text-action tw-w-3"
+				:icon="mdiBookmarkOutline"
+			/>
+		</button>
+		<button
+			v-if="isFavorite"
```

Comment:
I think you can use the `isFavorite` value to simplify this template to only a single button instance.
- remove v-if and use a ternary on the icon to switch between the icon type.
- also use a ternary in your analytics even to swap between `Favorite` and `Un-Favorite`.